tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -250,66 +250,77 @@ public void testHandleDescribe() {
             }
         });
         assertEquals(format("Feature: foo.bar\tSupportedMinVersion: 0\tSupportedMaxVersion: 10\tFinalizedVersionLevel: 5\tEpoch: 123%n" +
-            "Feature: metadata.version\tSupportedMinVersion: 3.3-IV0\tSupportedMaxVersion: 3.3-IV3\tFinalizedVersionLevel: 3.3-IV2\tEpoch: 123"), describeResult);
+            "Feature: metadata.version\tSupportedMinVersion: 3.3-IV3\tSupportedMaxVersion: 3.5-IV0\tFinalizedVersionLevel: 3.4-IV0\tEpoch: 123"), describeResult);
     }
 
     @Test
-    public void testHandleUpgrade() {
+    public void testHandleUpgradeToUnsupportedMetadataVersion() {
         Map<String, Object> namespace = new HashMap<>();
         namespace.put("metadata", "3.3-IV1");
```

Comment:
nit: This `3.3-IV1` variable also can add to `MetadataVersionTestUtils` class.

Reply:
I don't think we need to do that - the feature levels are opaque, but the metadata versions are clear and don't need a constant for usage in tests (they would need a constant within non test code but we don't use this in non test code).